### PR TITLE
runk: run kernel package

### PR DESCRIPTION
**Pull request by tiborvass**
This moves most of sbox into a simple runk package. Sbox thus becomes a main package.

### CODE REVIEW

**Comment by tiborvass** [comment]
@tonistiigi The Dockerfile can't work because the PR is not merged to master. Maybe we should just put sbox in runk/cmd/sbox. Or runk/cmd/runk ? lol

**Comment by tonistiigi** [comment]
@tiborvass You can put sbox into `cmd/sbox` and document that it needs to be built with `docker build -f ./cmd/sbox/Dockerfile .`

File: runk/Dockerfile [hunk 1]
```diff
@@ -3,6 +3,7 @@
 FROM golang:1.12-alpine AS sbox
```
Comment (tonistiigi) [inline runk/Dockerfile]: why did you move this under runk if it builds sbox.

File: runk/README.md [hunk 2]
```diff
@@ -1,3 +1,9 @@
+# runk
+
+## Example: sbox demo
```
Comment (tonistiigi) [inline runk/README.md]: same

File: runk/cmd/sbox/main.go [hunk 3]
```diff
@@ -0,0 +1,32 @@
+package main
```
Comment (tonistiigi) [inline runk/cmd/sbox/main.go]: if sbox is a subpackage of runk it doesn't make sense to have runk package at all, or the project should be called runk.

File: runk/README.md (hunk 2 above)
Comment (tiborvass) [reply runk/README.md]: So you want me to `docker build -f Dockerfile ../..` ?

Comment (tonistiigi) [reply runk/README.md]: `docker build -f ./cmd/sbox/Dockerfile .`

File: runk/Dockerfile (hunk 1 above)
Comment (tiborvass) [reply runk/Dockerfile]: My thinking was that it never hurts to have a Dockerfile for Go dev or test env, but I can put it in sbox, but not sure about README (see below).

File: runk/README.md (hunk 2 above)
Comment (tiborvass) [reply runk/README.md]: that doesn't work if I put the readme in cmd/sbox like I understood you asked in https://github.com/tonistiigi/vzor/pull/2#discussion_r281045027

Comment (tonistiigi) [reply runk/README.md]: it should be obvious where you are running it from the command. you can write it out if needed. Same thing for example for building dockerfile frontend in buildkit

**Review by tonistiigi — APPROVED** [review]
